### DISCUSSION TITLE
ctrlr Version Confusion

### DISCUSSION THREAD

**Discussion by Godlike-Productions** [RomanKubiak/ctrlr#162] [Q&A] [answered]
The latest version seems to be 5.6.1 here, but I'm running 6.0.26 in Windows, which is still available at nightly builds (https://ctrlr.org/nightly/?C=M;O=D).

Has there been a rollback for development?

For the record 6.0.26 has been the most stable of all the versions I've used and managed to rescue a panel that I thought I'd lost which kept crashing earlier versions of ctrlr.

**Comment by Possemo** [comment, marked as answer]
Yes, you see it right. I was confudsed too, Roman explained it somewhere but I don't know where exactly. Version numbering got back from 6.x to 5.5. The newest v5.6.x tries to make the transition to JUCE v6.x so it is still in early state. For me v5.5.9 has some problems too, so I suggest staying on 5.3.201 or 6.0.26 for now.

**Reply by Godlike-Productions** [reply]
Yeah, I'm not moving until I have finished this project, at least. I'm tantalizingly close.

**Reply by shooking** [reply]
As I said on another thread I got bitten by this a year ago. 6.x changed the APIs. I developed a few panels. Hit a snag. "You should use 5.31". Ok ... bummer much of my code no longer works.
I will keep saying this until someone kindly tells me how to checkout a specific version - but if you build from source as I do on the Pi it isnt clear what one is building.

**Reply by Possemo** [reply]
I could be one of the guys who recommended v5.3.201. For a long time 5.3.201 looked like the only choice as developement got stuck in about 2017. Upgrade to new JUCE versions introduced more bugs than features. It looks a bit different now. I think v5.6 could develop to a mature state and could become the new preferred choice. It's cool that some other C++ developers are contributing to the project.
